Source/SwiftLintFramework/Models/RuleIdentifier.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  RuleIdentifier.swift
+//  SwiftLint
+//
+//  Created by Frederick Pietschmann on 3/5/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum RuleIdentifier: Hashable, ExpressibleByStringLiteral {
+    case all
+    case some(identifier: String)
+
+    private static let allStringRepresentation = "all"
+
+    public var hashValue: Int {
+        switch self {
```

Comment:
what do you think about using `stringRepresentation.hashValue`?

Reply:
Makes sense